data_kennel/config.py

```diff
@@ -259,9 +260,6 @@ def get_monitors(self, tags=None):
                                                    notifications)
                     monitor['message'] = message
 
-                    # This isn't needed anymore
-                    del monitor['notify']
-
```

Comment:
Won't some monitoro still have a notify section?

Reply:
I saw the comment above, thought, it's not needed. Is it?

Reply:
Let's do it safe for now

Reply:
The comment doesn't mean that the del statement isn't needed, it means that the notify attribute is not needed anymore at that point. A bit of a difference. :D